### DISCUSSION TITLE
使用时浏览器会显示证书不安全

### DISCUSSION THREAD

**Discussion by Hgnim** [SpaceTimee/Sheas-Cealer#23] [Q&A] [answered]
我想知道是什么原因导致的证书无效，是否真的存在数据被拦截的风险。

**Comment by SpaceTimee** [comment, marked as answer]
浏览器显示不安全是因为 SNI server_name 和 服务器证书域名 不匹配 (因为 SNI 被 Sheas Cealer 改了)，实际上由于 Sheas Cealer 通过浏览器启动参数指定了忽略证书错误，浏览器依然会正常使用服务器证书来进行 TLS 握手，后续的通信也都是 https 连接的，中间人不可解密，不会增大数据被拦截的风险
